datalad/downloaders/s3.py

```diff
@@ -60,32 +52,46 @@ class S3Authenticator(Authenticator):
     allows_anonymous = True
     DEFAULT_CREDENTIAL_TYPE = 'aws-s3'
 
-    def __init__(self, *args, host=None, **kwargs):
+    def __init__(self, *args, host=None, region=None, **kwargs):
```

Comment:
This was removed by @mslw in the previous PR. My guess is because host did not have an obvious counterpart. I think it would be sensible to restore it if `endpoint_url` is the equivalent concept (modulo some shim?).

Reply:
gotcha... insofar it seems that host was just a FQDN (e.g. s3.us-east-2.amazonaws.com) whenever `endpoint_url` should be a URL thus have protocol (https) added... So I will reincarnate `host` and extend it to start with protocol (https) if doesn't have `://` in it, and provide as `endpoint_url`.  Sounds ok'ish?

Reply:
implemented although without tests